docs/advanced/integrations/jenkins.md

```diff
@@ -0,0 +1,48 @@
+With Jenkins it is possible to do scriped pipelines but also declarative pipelines. I prefere declarative pipelines. You will find more information about pipelines in jenkins at https://www.jenkins.io/doc/book/pipeline/.
+
+The result parser is included in an existing plugin called Warning NG (https://plugins.jenkins.io/warnings-ng/).
+
+Ensure that the plugin is installed, than you need to execute the scan in the shell
+```
+sh "trivy image -f json -o results.json aquasec/trivy:0.19.2"
+```
+Now you cold collect the results immediatly or doing this in a post section on the stage or a post section at the end of he pipeline.
+```
+recordIssues(tools: [trivy(pattern: 'results.json')])
+```
+
+A simple example of a declarative pipeline in a kubernetes envionment:
```

Comment:
I couldn't test with successful this pipeline. I tried this and works:

```
timestamps {
	node(){
    stage('install trivy') {
      sh "curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin v0.21.0"
    }
        
    stage('scan with trivy') {
      sh "trivy image -f json -o results.json nginx:1.18"
      recordIssues enabledForFailure: true, failOnError: true, aggregatingResults: true, tool: trivy(pattern: 'results.json')
    }
   }
}
```

Reply:
@tofuatjava check the results using this sample

<img width="1767" alt="Screen Shot 2021-11-23 at 19 15 49" src="https://user-images.githubusercontent.com/8355621/143137761-46e60cb4-4b14-4ad7-85b8-d1ef6e0eabe0.png">